util/flags.py

```diff
@@ -111,7 +111,7 @@ def create_flags():
 
     # Initialization
 
-    tf.app.flags.DEFINE_integer ('random_seed',      4567,        'default random seed that is used to initialize variables')
+    tf.app.flags.DEFINE_integer ('random_seed',      4568,        'default random seed that is used to initialize variables')
```

Comment:
Git commit says "75 epochs" but `tc-train-tests.sh` still uses 105 ?

Reply:
That's probably why it wasn't failing so often. 75 is from run-ldc93s1.sh